lib/backends/system-manager-backend.js

```diff
@@ -49,13 +51,34 @@ class SystemManagerBackend extends KVBackend {
       client = this._clientFactory(factoryArgs)
     }
     try {
-      const data = await client
-        .getParameter({
-          Name: key,
-          WithDecryption: true
-        })
+      const params = {
+        Name: key,
+        WithDecryption: false
+      }
+      const check = await client
```

Comment:
You assign `check` once but subsequent calls with `NextToken` return an array of parameter metadata that must be merged